packages/ui/src/components/Component.svelte

```diff
@@ -120,6 +120,8 @@
           on:validate
           on:submit
           on:select
+          on:keydown
```

Comment:
these are too specific, please try to avoid using them, especially `editingEnded`